src/run-on-arch.sh

```diff
@@ -35,9 +37,7 @@ install_deps () {
   # Install support for non-x86 emulation in Docker via QEMU.
   # Platforms: linux/arm64, linux/riscv64, linux/ppc64le, linux/s390x,
   #            linux/386, linux/arm/v7, linux/arm/v6
-  sudo apt-get update -q -y
-  sudo apt-get -qq install -y qemu qemu-user-static
-  docker run --rm --privileged multiarch/qemu-user-static --reset -p yes --credential yes
+  docker run --rm --privileged $QEMU_STATIC_CONTAINER --reset -p yes --credential yes
```

Comment:
I think this answers my earlier question. It won't work with QUS because it uses different options' names (https://dbhi.github.io/qus/tests.html)